driving_gridworld/obstacles.py

```diff
@@ -49,3 +51,22 @@ def expected_reward_for_collision(self, speed):
 
     def __str__(self):
         return 'p'
+
+
+class CarObstacle(Obstacle):
+    def __init__(self, row, col, prob_of_appearing=0.2, obst_speed=1):
+        self.row = row
```

Comment:
Since the body of this constructor should be identical to its parent's, you should just call `super().__init__(*args, speed=speed, **kwargs)`. The constructor arguments can also be changed to `def __init__(self, *args, speed=1, **kwargs)`, since `CarObstacle` doesn't actually need to know what the arguments are to its parent's constructor, except for `speed`.